app/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="purple_200">#FFBB86FC</color>
-    <color name="purple_500">#FF6200EE</color>
-    <color name="purple_700">#FF3700B3</color>
+    <color name="dark_red">#732020</color>
```

Comment:
I saw that the color scheme in the storyboard did not match the one in the project. Therefore, I found the right color
using Adobe color finder, and changed in the code. It looks really nice with dark red instead of purple.